spec/wayfinders/wayfinder_spec.rb

```diff
@@ -0,0 +1,703 @@
+# frozen_string_literal: true
+require "rails_helper"
+
+RSpec.describe Wayfinder do
+  context "when given a ScannedResource" do
```

Comment:
this context section is quite large; it seems to encompass more than just scanned resources stuff.